translate/src/modules/translationform/components/TranslationForm.css

```diff
@@ -81,6 +74,46 @@
 }
 
 .translationform .accesskeys .key.active,
-.translationform .accesskeys .key:hover {
+.translationform .accesskeys .key:hover:not(:disabled) {
   border-color: #7bc876;
 }
+
+.translationform .accesskeys .key:disabled {
+  cursor: default;
+}
+
+.cm-editor {
```

Comment:
CodeMirror packages in some styling for itself via CSS-in-JS; these are overrides on those defaults. It does include its own [styling system](https://codemirror.net/examples/styling/), but relying on CSS is just as valid, and more in line with what we've been doing so far.